apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/EventActivity.java

```diff
@@ -217,13 +217,14 @@ private boolean onlyStringProperties() {
 
     @SuppressWarnings("unused")
     public void send(@SuppressWarnings("UnusedParameters") View view) {
+        AnalyticsTransmissionTarget target = getSelectedTarget();
         PersistenceFlag persistenceFlag = PersistenceFlag.values()[mPersistenceFlagSpinner.getSelectedItemPosition()];
         int flags = getFlags(persistenceFlag);
         String name = mName.getText().toString();
         Map<String, String> properties = null;
         EventProperties typedProperties = null;
         if (mProperties.size() > 0) {
-            if (onlyStringProperties() && persistenceFlag == PersistenceFlag.DEFAULT) {
+            if (onlyStringProperties() && (persistenceFlag == PersistenceFlag.DEFAULT || target == null)) {
```

Comment:
Discussed this with @elamalani this should also be available for `AnalyticsTransmissionTarget`.